app/src/main/java/com/project/squirrelobserver/squirrelObserver/MainActivity.java

```diff
@@ -182,9 +188,11 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                                         settingsPath, GlobalVariables.settingsLocationTag, GlobalVariables.locationCSVPath);
 
                                 // Set Label
-                                locationLabel.setText(
-                                        GlobalVariables.locationCSVPath.substring(
-                                                GlobalVariables.locationCSVPath.lastIndexOf("/") + 1));
+                                if(fileName == "") {
```

Comment:
needs to be written as "".equals(fileName)
== is for checking memory addresses, it tells if the two objects are literally the exact same object.
.equals is for checking equality within two separate objects.

For example:
Integer a = new Integer(5);
Integer b = new Integer(5);
if (a == b)
  return true;
else if (a.equals(b))
  return false;
// This will return false because the memory addresses are not the same.